src/report.rs

```diff
@@ -44,7 +46,50 @@ impl Report {
       format!("unix:{:?}", platform.os_type).into()
     };
 
-    let backtrace = format!("{:#?}", Backtrace::new());
+    //We skip non-user code frames, including Backtrace::new()
+    const SKIP_FRAMES_NUM: usize = 8;
+    //Code is based on backtrace source
+    const HEX_WIDTH: usize = mem::size_of::<usize>() * 2 + 2;
+
+    let mut backtrace = String::new();
+
+    for (idx, frame) in Backtrace::new()
```

Comment:
This loop does a lot of stuff. Documenting a little about what it does/ is _supposed_ to do is gonna be very helpful to anyone reading the code in a few months time :)